decidim-elections/app/packs/src/decidim/elections/admin/trustees_process.js

```diff
@@ -14,10 +14,14 @@ $(async () => {
   const bulletinBoardClient = new Client({
     apiEndpointUrl: $trusteesProcess.data("apiEndpointUrl")
   });
+  const typesFilter = ["create_election", processType];
+  if (processType === "tally_started") {
+    typesFilter.push("tally");
```

Comment:
When the process type is `tally_started`, we also need to include all the messages marked as `tally.nnn`.

This is because these keys exist internally in the bulletin board app (and its voting schemes):
https://github.com/decidim/decidim-bulletin-board/blob/3da82f8e90bc78f75febcb70d3e276114c429884/voting_schemes/dummy/js-adapter/src/trustee_wrapper.js#L13-L16

They also exist in multiple other places e.g. within the python-wrapper and the ruby client.

In the past, this has assumed that the "type" prefix for the message type matches always the current step key. This is no longer the case because I did not want to touch those message keys in the bulletin board and its voting schemes.

This can be one thing we can look into in future versions of the bulletin board app and its related packages.

Reply:
Actually I figured we don't need this change if we just pass "tally" as the process type filter from the view: 370873fc9f7c96a7c682a563f1003aa6af459f3a